etc/inc/disable-common.inc

```diff
@@ -495,7 +495,9 @@ blacklist ${PATH}/xev
 blacklist ${PATH}/xinput
 # from 0.9.67
 blacklist ${PATH}/ssh
-blacklist /usr/lib/openssh/ssh-keysign
```

Comment:
`/usr/lib/openssh/ssh-keysign` is the path in Debian.
Please keep it. :-)

Reply:
ah, the directory is now blacklisted.
but then why not blacklist the whole `/usr/libexec/openssh` as well?